src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -310,14 +310,15 @@ public static string MachineName
 
         public static string NewLine => "\n";
 
-        private static Lazy<OperatingSystem> s_osVersion = new Lazy<OperatingSystem>(() =>
+        private static Lazy<OperatingSystem> s_osVersion = new Lazy<OperatingSystem>(GetOperatingSystem(Interop.Sys.GetUnixRelease()));
```

Comment:
Sorry that I missed this on the first review, but this should be passing `() => GetOperatingSystem(Interop.Sys.GetUnixRelease())` to the Lazy constructor so that it's actually lazy, right?

That new constructor makes this mistake easy to make and miss on review. :(